packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -399,6 +399,16 @@ class FileCollection extends DocumentCollection {
     forceFileDownload(`${href}?Dl=1`, filenameToUse)
   }
 
+  /**
+   * Get the binary of a file or a specific version of a file
+   * used for Mobile native download
+   *
+   * @param {string} id Id of the io.cozy.files or io.cozy.files.version
+   *
+   */
+  async getFileContent(id) {
```

Comment:
I think it would be clearer to use "get" only when the data is local, here since we need to fetch it, I'd prefer fetchFileContent, it makes it clear to the caller that an asynchronous call is going to be done.